RELEASE.md

```diff
@@ -1,3 +1,14 @@
+# Release 0.17.7
```

Comment:
Will this exist? 🤔

Reply:
Yes I think it makes sense to make a small release before `0.18.0`